src/libs/actions/IOU.ts

```diff
@@ -133,15 +232,17 @@ Onyx.connect({
 
 /**
  * Initialize money request info
- * @param {String} reportID to attach the transaction to
- * @param {Boolean} isFromGlobalCreate
- * @param {String} [iouRequestType] one of manual/scan/distance
+ * @param reportID to attach the transaction to
+ * @param iouRequestType one of manual/scan/distance
  */
-function startMoneyRequest_temporaryForRefactor(reportID, isFromGlobalCreate, iouRequestType = CONST.IOU.REQUEST_TYPE.MANUAL) {
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Maybe apply this rule once for the entire file?

Reply:
@rezkiy37 I did it initially, but was asked to remove it, see [comment](https://github.com/Expensify/App/pull/34092#discussion_r1448928119)